### PR TITLE
Implement reset camera parameters

### PR DESCRIPTION
**Pull request by SaliniVenate**
Upon receiving MAV_CMD_RESET_CAMERA_SETTINGS, csd should reset those camera parameters which can be set.

### CODE REVIEW

File: src/mavlink_server.cpp [hunk 1]
```diff
@@ -518,6 +518,20 @@ void MavlinkServer::_handle_param_ext_set(const struct sockaddr_in &addr, mavlin
     }
 }
 
+void MavlinkServer::_handle_reset_camera_settings(const struct sockaddr_in &addr,
+                                                  mavlink_command_long_t &cmd)
+{
+    bool success = false;
+
```
Comment (lbegani) [inline src/mavlink_server.cpp]: Add a check for param index 1

File: src/CameraDeviceV4l2.cpp [hunk 2]
```diff
@@ -129,6 +130,69 @@ int CameraDeviceV4l2::getMode()
...
+        case V4L2_CID_SHARPNESS:
+            camParam.setParameter(CameraParameters::SHARPNESS, (uint32_t)queryctrl.default_value);
+            break;
+            //     case V4L2_CID_AUTOGAIN:
+            //     case V4L2_CID_HFLIP:
+            //     case V4L2_CID_VFLIP:
+        }
+    }
+    log_debug("Reset Done!");
+    return 0;
```
Comment (lbegani) [inline src/CameraDeviceV4l2.cpp]: Return appropriate return value instead of default success

File: src/CameraComponent.cpp [hunk 3]
```diff
@@ -302,6 +302,12 @@ int CameraComponent::setCameraMode(uint32_t mode)
     return 0;
 }
 
+int CameraComponent::resetCameraSettings()
+{
+    mCamDev->resetParams(mCamParam);
+    return 0;
```
Comment (lbegani) [inline src/CameraComponent.cpp]: Return appropriate return value instead of default success

**Comment by SaliniVenate** [comment]
Made changes as per review comments.

**Pull request merged**